specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2019-01-01-preview/SecurityInsights.json

```diff
@@ -770,6 +770,454 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/{operationalInsightsResourceProvider}/workspaces/{workspaceName}/providers/Microsoft.SecurityInsights/cases/{caseId}/relations": {
+      "get": {
+        "x-ms-examples": {
+          "Get all case relations.": {
+            "$ref": "./examples/cases/relations/GetAllCaseRelations.json"
+          }
+        },
+        "tags": [
+          "CaseRelations"
+        ],
+        "description": "Gets all case relations.",
+        "deprecated": true,
```

Comment:
>"deprecated": true, [](start = 8, length = 19)

You're adding new paths that are deprecated? If this is intentional, can you explain?

Reply:
(The comment applies to operations below as well.)

---
In reply to: [324915817](https://github.com/Azure/azure-rest-api-specs/pull/7191#discussion_r324915817) [](ancestors = 324915817)

Reply:
to be consistent with /cases also marked as such.

Reply:
Discussed deprecated in person. The API will be renamed in another API version.

---
In reply to: [324916053](https://github.com/Azure/azure-rest-api-specs/pull/7191#discussion_r324916053) [](ancestors = 324916053,324915817)